install-to-localhost.sh

```diff
@@ -3,6 +3,10 @@
 # get working dir
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
+echo $DIR
+
+echo "Pulling Submodules --------------------------------->"
```

Comment:
@knutole Will remove "------>"  . I added this as when a lot of logs are in the terminal , hard to see which is what .